most-run/package.json

```diff
@@ -28,27 +28,25 @@
   "homepage": "https://cycle.js.org",
   "dependencies": {
     "@cycle/run": "4.x",
-    "xstream": "*",
-    "most": "*"
+    "xstream": "*"
```

Comment:
Uh, why remove the `most` dependency from `most-run`?

Reply:
Error on my side